### ISSUE TITLE
Add Arabic names

### ISSUE THREAD

**Issue by BenOvermyer** [ironarachne/namegen#56]
Add Arabic names to the project.

Note: this will require special handling to the structure of Arabic names. Compare to how the Icelandic name generator is coded.

**Comment by BenOvermyer** [comment]
This may be useful: https://github.com/zakahmad/ArabicNameGenderFinder

**Comment by DireWolf707** [comment]
Assign this issue if not taken, so i can start working on it ASAP.

**Comment by BenOvermyer** [comment]
Have fun with this one! It'll require some custom logic.

**Comment by DireWolf707** [comment]
Do want names like أبدع or in plain English

**Comment by BenOvermyer** [comment]
If you could do _both_ and offer an argument to switch between them, that would be ideal. Otherwise, either Arabic or English script is fine.

**Comment by BenOvermyer** [comment]
@DireWolf707 Are you going to do this one this year, or should I let someone else have it?

**Comment by DireWolf707** [comment]
I was unable to do this is issue.
Sry about that, unassigning myself
But will be looking forward to work on other issues of this repo

**Comment by ArshErgon** [comment]
Arabic is pretty hard language, if we translate the Arabic to English its beauty will be lost + its pronunciation too, Abraham in English, Ibraham in Arabic. can we use both? like Arabic and English name by using a dictionary / hash map.

**Comment by BenOvermyer** [comment]
@ArshErgon Sure. If it's best represented in Arabic, though, then we should keep it in Arabic!.
